cddl/intrep-ev.cddl

```diff
@@ -0,0 +1,12 @@
+ev = [
```

Comment:
It seems endorsements, conditional endorsements, and conditional series endorsements are unrolling a recursive definition that would allow for simpler expression.

```
evs = [ + {
? / condition: / 0 => [ + ECT ]
/ addition: / 1 => evs / [ + ECT ]
} ]
```
If a condition applies and the addition isn't a map, add the ECTs. If the addition is a map, re-enter the top of the loop to further specialize the condition. If the condition doesn't apply, go to the next map in the series.

This allows for nested if/elseif.../else expressions. I don't think the nonempty selection requirement allows for a bare "else".

Reply:
No, [ + ECT ] is a base case. You can unfold as many nestings as you want, but you eventually bottom out in a flat list of additions.

Reply:
@deeglaze clarifying question: your `evs` definition above would also subsume `ev`?